benchmarks/arithmetic_benchmark.py

```diff
@@ -8,7 +8,7 @@
 import os
 import modin.pandas as pd
 
-from utils import time_logger
+from .utils import time_logger
```

Comment:
This import won't work since the directory is not set up with an `__init__.py` script.